pkgs/applications/misc/kiwix/default.nix

```diff
@@ -35,6 +36,7 @@ mkDerivation rec {
     qtwebengine
     qtsvg
     qtimageformats
+    wrapQtAppsHook
```

Comment:
This should be in `nativeBuildInputs`, according to the manual.

Reply:
Thanks for the review! Amended my commit.